src/api/app/models/package.rb

```diff
@@ -956,9 +956,14 @@ def jobhistory_list(project, repository, arch, limit = 100)
                                      limit: limit)
 
     local_jobs_history = []
-    results.elements('jobhist').reverse_each do |result|
+    results.elements('jobhist').each_with_index do |result, i|
+      prev_result = results.elements('jobhist')[i - 1]
+      prev_verifymd5 = prev_result.nil? ? nil : prev_result['verifymd5']
```

Comment:
[nitpick] 
``
prev_verifymd5 = results.elements('jobhist')[i - 1].try(:fetch, 'verifymd5', nil)
``

I would also consider in just setting the previous result in each run at the end instead of going back in every iteration?